diff --git a/node_modules/react-native-gesture-handler/src/utils.ts b/node_modules/react-native-gesture-handler/src/utils.ts
index 085c8c7..eff5980 100644
--- a/node_modules/react-native-gesture-handler/src/utils.ts
+++ b/node_modules/react-native-gesture-handler/src/utils.ts
@@ -1,6 +1,4 @@
-import pack from 'react-native/package.json';
-
-const [majorStr, minorStr] = pack.version.split('.');
+const [majorStr, minorStr] = '2.12.0'.split('.');
 export const REACT_NATIVE_VERSION = {
   major: parseInt(majorStr, 10),
   minor: parseInt(minorStr, 10),
